.travis.yml

```diff
@@ -37,14 +37,12 @@ install:
   - source activate skogestad
   - pip install codeclimate-test-reporter
   - conda env export  # debugging
-# TODO: Package
-#  - python setup.py install
+# Package: Robustcontrol
+  - pip install -e .
 
 # Run test
 script:
   - coverage run test_all_notebooks.py
-  - coverage run -a utils.py
```

Comment:
I think you should figure out how to still run these doctests. The notebooks and the utils are two different sets of tests.